cupy/cuda/cufft.pyx

```diff
@@ -524,27 +552,31 @@ class Plan1d(object):
             share = self.batch_share
 
             if action == 'scatter':
-                if isinstance(a, cupy.ndarray):
-                    for i in range(nGPUs):
-                        count = int(share[i] * self.nx)
+                # if isinstance(b, cupy.ndarray):
```

Comment:
Please remove this comment and fix indentation.

Reply:
Done.